classes/02class/exercises/c02-network07/starkmatt/c02-network07.txt

```diff
@@ -0,0 +1,31 @@
+##Exercise 7
+- VPC Dashboard service, and on the left hand side select Route Tables
+
+- Then select my Public RT, and added the following 0.0.0.0/0 | target devopsacademy-igw igw-080859d0d893c8a00
+
+- After associating, I then selected my private route table added the same details as above but I selected the NAT Gateway as the target | devopsacademy-ngw nat-07d1ac4edb3e79d4b
+
+- After creating RT's, I then went into Subnet Associations and selected Private Subnets for the Private RT, and Public Subnets for the Public RT
+
+Why did you add 0.0.0.0 route to the IGW on public route table?
+0.0.0.0 was used so it covers awhole range of addresses so they're able to access the Public Instance
```

Comment:
That's correct.
FYI, in a route table the order of the rules matters. So if you put this `0.0.0.0` rule as the first one, it will catch any traffic and send it to the IGW.

Reply:
Not entirely sure @denstorti . With route tables, the order of route entries doesn't really matters. It will actually try to match the most specific route first. 

From AWS: https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html

If your route table has multiple routes, we use the most specific route that matches the traffic (longest prefix match) to determine how to route the traffic.